model_data_preprocess/postprocessing_E3SM_data_for_TC_analysis.sh

```diff
@@ -0,0 +1,100 @@
+
+#For typical EAM v2 ne gp2 grids.
+start="0051"
+end="0060"
+caseid="20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis"
+drc_in=/global/cscratch1/sd/forsyth/e3sm_unified_test_zppy/${caseid}/archive/atm/hist
+# Warning: because somehow tempest-remap only can writes grid file on SCRATCH space. The resulted files will be moved to another path at the end.
+result_dir_fin=/global/cfs/cdirs/e3sm/e3sm_diags/postprocessed_e3sm_v2_data_for_e3sm_diags/${caseid}/
+mkdir -p $result_dir_fin
+result_dir=$SCRATCH/tests/tc-analysis/
+
+
+#res = 30/120 for ne30/ne120 grids
+res=30               # res = 30/120 for ne30/ne120 grids
+pg2=true             # Set false for v1 production simulations
+atm_name="eam"       # Use "cam" for v1 production simulations
+
+
+##For typical v1 ne SE grids.
+#start="1880"
+#end="1881"
+#caseid="20180215.DECKv1b_H1.ne30_oEC.edison"
+#drc_in=/global/cfs/cdirs/e3smpub/E3SM_simulations/${caseid}/archive/atm/hist
+#result_dir=/global/cfs/cdirs/e3sm/e3sm_diags/postprocessed_e3sm_v1_data_for_e3sm_diags/${caseid}/tc-analysis/
+#
+##res = 30/120 for ne30/ne120 grids
+#res=30               # res = 30/120 for ne30/ne120 grids
+#pg2=false             # Set false for v1 production simulations
+#atm_name="cam"       # Use "cam" for v1 production simulations
+
+
+
+mkdir -p $result_dir
+file_name=${caseid}_${start}_${end}
+
+# Generate mesh files (.g).
+#/global/homes/p/paullric/tempestremap/bin/GenerateCSMesh --res $res --alt --file ${result_dir}outCSne$res.g
+GenerateCSMesh --res $res --alt --file ${result_dir}outCSne$res.g
+out_type="CGLL"
+# For v2 production simulation with pg2 grids:
+if $pg2; then
+    GenerateVolumetricMesh --in ${result_dir}outCSne$res.g  --out ${result_dir}outCSne$res.g --np 2 --uniform
+    out_type="FV"
+fi
+echo $out_type
+# Generate connectivity files (.dat)
+#/global/homes/p/paullric/tempestextremes/bin/GenerateConnectivityFile --in_mesh ${result_dir}outCSne$res.g --out_type CGLL --out_connect ${result_dir}connect_CSne${res}_v2.dat
+GenerateConnectivityFile --in_mesh ${result_dir}outCSne$res.g --out_type $out_type --out_connect ${result_dir}connect_CSne${res}_v2.dat
+
+# Get the list of files
+cd ${drc_in};eval ls ${drc_in}/${caseid}.$atm_name.h2.*{${start}..${end}}*.nc >${result_dir}inputfile_${file_name}.txt
+cd ${drc_in};eval ls ${caseid}.$atm_name.h2.*{${start}..${end}}*.nc >${result_dir}outputfile_${file_name}.txt
+#cd ${drc_in};eval ls ${drc_in}/${caseid}.cam.h2.*{${start}..${end}}*.nc >${result_dir}inputfile_${file_name}.txt
+#cd ${drc_in};eval ls ${caseid}.cam.h2.*{${start}..${end}}*.nc >${result_dir}outputfile_${file_name}.txt
+
+cd ${result_dir}
+# Detection threshold including:
+# The sea-level pressure (SLP) must be a local minimum; SLP must have a sufficient decrease (300 Pa) compared to surrounding nodes within 4 degree radius; The average of the 200 hPa and 500 hPa level temperature decreases by 0.6 K in all directions within a distance of 4Â° from the location o fSLP minima
```

Comment:
`4Â°` -- is that correct or a formatting error/typo?

Reply:
should replace with 4 degree radius.